mindmeld/ser.py

```diff
@@ -133,6 +133,7 @@ def parse_numerics(
     locale=None,
     time_zone=None,
     timestamp=None,
+    url=None,
```

Comment:
You added a URL param here but the caller to this function has not been changed.

Reply:
Hmm I thought about not exposing the URL to the caller, but you are right. The caller should have the option to expose the URL as well.